src/main/java/net/fabricmc/loader/FabricLoader.java

```diff
@@ -253,7 +253,7 @@ protected void finishModLoading() {
 		// add mods to classpath
 		// TODO: This can probably be made safer, but that's a long-term goal
 		for (ModContainer mod : mods) {
-			if (!mod.getInfo().getId().equals("fabricloader")) {
+			if (!mod.getInfo().getId().equals("fabricloader") && !mod.getInfo().getId().equals("java")) {
```

Comment:
Possibly should do `!"builtin".equals(mod.getMetadata().getType())` instead, at least from the impression that built in mods will already be on the classpath in some way

Reply:
Humm, yeah... I’ll have to test that shouldn’t cause any issues

Reply:
I believe checking for builtin would be the best choice, the fallout on MC (the other builtin mod) needs testing though.

Reply:
Done, tested in dev and pod with AOF seems to be fine.